### PR TITLE
add Coder hosted getting started guide

### PR DESCRIPTION
**Pull request by ericpaulsen**
this PR includes context around what is Coder's hosted beta, and the initial steps needed to access a Coder hosted deployment. it stops short of stepping through the workspace provider configuration, as I expect this to be a separate guide. [see clubhouse ticket](https://app.clubhouse.io/coder/story/16428/docs-request-adding-a-workspace-provider-and-cluster).

[ch16426]

### CODE REVIEW

**Comment by bpmct** [comment]
> it stops short of stepping through the workspace provider configuration, as I expect this to be a separate guide.

I will be sending a PR for this today!

File: guides/saas/getting-started.md
Comment (bpmct) [inline guides/saas/getting-started.md:9]: @CoderErik created a lovely graphic demonstrating how this works. thoughts on adding a version of this in a later revision?

We also discussed adding a second graphic to demonstrate how the clients actually have a point-to-point connection with workspaces. Thoughts on that as well?

@jmilling33 could probably work his magic on it

[Shared Hosting.pdf](https://github.com/cdr/docs/files/7078351/Shared.Hosting.pdf)

File: guides/saas/getting-started.md
Comment (bpmct) [inline guides/saas/getting-started.md:19]: I suggest adding a CTA for folks to try it, as we do on out home page

```suggestion
If you interested in trying the limited beta, [contact us](https://coder.com/contact?note=I%20would%20like%20to%20try%20the%20hosted%20offer.%0A%0ANumber%20of%20developers%3A%0A%0AUse%20case%3A))

Curious how this works? Here's a breakdown of the architecture:
```

File: guides/saas/getting-started.md
Comment (bpmct) [inline guides/saas/getting-started.md:43]: These details are currently emailed to the admin. Perhaps we should mention that here in case they forget what their URL or password is?

File: guides/saas/getting-started.md
Comment (bpmct) [reply guides/saas/getting-started.md:19]: JFYI: this probably isn't linted correctly. i did from the web ui

File: guides/saas/getting-started.md
Comment (bpmct) [reply guides/saas/getting-started.md:9]: whoops here is the updated graphic

[Shared Hosting (1).pdf](https://github.com/cdr/docs/files/7078391/Shared.Hosting.1.pdf)

Comment (ericpaulsen) [reply guides/saas/getting-started.md:9]: Definitely think @CoderErik's graphic would fit well in here.

File: guides/saas/getting-started.md
Comment (ericpaulsen) [reply guides/saas/getting-started.md:19]: Good call - will add.

File: guides/saas/getting-started.md
Comment (ericpaulsen) [reply guides/saas/getting-started.md:43]: You're right - in the current state, it assumes the reader knows they've been emailed the credentials. I can clarify.

File: guides/saas/getting-started.md
Comment (CoderErik) [reply guides/saas/getting-started.md:9]: Here's a link to png version
https://lucid.app/publicSegments/view/7e950228-6b5a-47eb-b9d4-ed5a82e5898d/image.png

File: guides/saas/getting-started.md
Comment (mterhar) [inline guides/saas/getting-started.md:16]:
```suggestion
- DNS or TLS configuration
```

They still have to `install` a provider so I'm not sure "no installatino" is valid.

File: guides/saas/getting-started.md
Comment (mterhar) [inline guides/saas/getting-started.md:32]:
```suggestion
```

Unnecessary and subject to change as we scale.

File: guides/saas/getting-started.md
Comment (mterhar) [inline guides/saas/getting-started.md:49]:
```suggestion
   - Username should be the email address you provided
```

File: guides/saas/getting-started.md
Comment (mterhar) [inline guides/saas/getting-started.md:57]:
```suggestion
You will receive the credentials from Coder when the instance is provisioned.
When you log in, you'll be prompted to change your temporary password
```

We set up the license for them.

**Review by mterhar — CHANGES_REQUESTED** [review]

File: guides/saas/getting-started.md
Comment (ericpaulsen) [reply guides/saas/getting-started.md:9]: @jmilling33 from a design standpoint, should we defer to you on that? @CoderErik's graphic is accurate, but I just want to make sure we keeps things consistent.

File: guides/saas/getting-started.md
Comment (khorne3) [inline guides/saas/getting-started.md:56]: gotta remember to update this placeholder

**Comment by khorne3** [comment]
@ericpaulsen the naming is wip -- it is hosted beta atm, so I updated the URL accordingly

File: guides/saas/getting-started.md
Comment (jmilling33) [reply guides/saas/getting-started.md]: I am happy to restyle the graphic to make sure it's consistent with our branding. When do you need it by?

Comment (CoderErik) [reply guides/saas/getting-started.md]: @jmilling33 Just shared the original file with you. Check email for the link.

Comment (bpmct) [reply guides/saas/getting-started.md]: @jmilling33 no hard deadline. It would be nice to update this doc in ~1 week or so via another PR.

For now, we can leave the illustration out

**Comment by khorne3** [comment]
@ericpaulsen I updated the final bit of instructions w/ the links; what else do you think needs to be done before shipping?

**Review by khorne3 — APPROVED** [review]

File: guides/saas/getting-started.md
Comment (ericpaulsen) [reply guides/saas/getting-started.md]: before we merge - I think we need to update this to link to #576.

**Comment by ericpaulsen** [comment]
> @ericpaulsen I updated the final bit of instructions w/ the links; what else do you think needs to be done before shipping?

@khorne3 outside of linking to #576 at the bottom I think we're good. I see you force-pushed my branch to another commit. Will this prevent me from making another commit to my branch? thanks.

**Comment by khorne3** [comment]
@ericpaulsen I think it you pull the changes first, then edit, you should be good to go

**Comment by ericpaulsen** [comment]
> @ericpaulsen I think it you pull the changes first, then edit, you should be good to go

got it - all looks good to me! I'll click the big green button.

**Pull request merged**
